concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1286,6 +1286,10 @@ jobs:
                 . terraform-variables/cf-secrets.tfvars.sh
 
                 cp cf-tfstate/cf.tfstate updated-tfstate/cf.tfstate
+                terraform import aws_s3_bucket_lifecycle_configuration.droplets-s3 ((deploy_env))-cf-droplets
```

Comment:
Do we need to undo this change after the state file has been changed?

Reply:
Nope